src/Microsoft.AspNet.Hosting.WindowsServices/WebHostExtensions.cs

```diff
@@ -6,37 +6,37 @@
 namespace Microsoft.AspNet.Hosting.WindowsServices
 {
     /// <summary>
-    ///     Extensions to <see cref="IWebApplication" for hosting inside a Windows service. />
+    ///     Extensions to <see cref="IWebHost" for hosting inside a Windows service. />
     /// </summary>
-    public static class WebApplicationExtensions
+    public static class WebHostExtensions
```

Comment:
Our own Extension classes as well. Having the same namespace in different assemblies makes it hard to search for docs.